torch/autograd/profiler.py

```diff
@@ -486,6 +486,70 @@ def _call_end_callbacks_on_future(self, fut: Future[Any]) -> Future[Any]:
         return profiled_future
 
 
+class emit_itt(object):
```

Comment:
`torch.autograd.profiler` is a deprecated namespace. New development should be in `torch.profiler.`

Reply:
As `emit_nvtx` is still in `torch.autograd.profiler`, `emit_itt` aligns with `emit_nvtx`.